core/src/main/java/org/apache/carbondata/core/locks/HdfsFileLock.java

```diff
@@ -39,16 +39,18 @@
    */
   private String location;
```

Comment:
Is this required anymore?

Reply:
I change this to lockFileDir, it need this parameter to mkdir.